app/src/main/java/de/danoeh/antennapod/adapter/NavListAdapter.java

```diff
@@ -119,7 +123,7 @@ private Drawable getDrawable(String tag) {
 
     @Override
     public int getCount() {
-        return getSubscriptionOffset() + itemAccess.getCount();
+        return getSubscriptionOffset() ;//+ itemAccess.getCount(); //Avoid feed
```

Comment:
I believe this peace of code is the reason I do not see any feeds in the drawer anymore